src/aws.js

```diff
@@ -27,9 +29,18 @@ function buildUserDataScript(githubRegistrationToken, label) {
       'tar xzf ./actions-runner-linux-${RUNNER_ARCH}-2.299.1.tar.gz',
       'export RUNNER_ALLOW_RUNASROOT=1',
       `./config.sh --url https://github.com/${config.githubContext.owner}/${config.githubContext.repo} --token ${githubRegistrationToken} --labels ${label}`,
-      './run.sh',
     ];
   }
+  if (config.input.runAsUser) {
+    userData.push(`chown -R ${config.input.runAsUser} .`);
+  }
+  if (config.input.runAsService) {
+    userData.push(`./svc.sh install ${config.input.runAsUser || ''}`);
+    userData.push('./svc.sh start');
+  } else {
+    userData.push(`${config.input.runAsUser ? `su ${config.input.runAsUser} -c` : ''} ./run.sh`);
```

Comment:
if `run.sh` supports only running as root then why do we run it under regular user? and if it supported the description of `run-runner-as-user` parameter should be updated.

and if this is possible why do we need to run it as a service?

also, won't it better to create the user within this script? also add it to sudoers to be closer to github environment?